app/src/views/Game.vue

```diff
@@ -226,6 +238,21 @@ export default {
 
         this.peekCountDown = setInterval(timer, 1000);
       }
+    },
+    isSubmitting(newValue, oldValue) {
+      console.log("is submitting", newValue);
+      if (!newValue) clearInterval(this.submitCountDown);
+      if (newValue && newValue !== oldValue) {
```

Comment:
Is this check necessary ?

Reply:
No, I refactored the watcher for more clarity.